superset/utils/screenshots.py

```diff
@@ -195,12 +195,34 @@ def resize_image(
 class ChartScreenshot(BaseScreenshot):
     thumbnail_type: str = "chart"
     element: str = "chart-container"
-    window_size: WindowSize = (800, 600)
-    thumb_size: WindowSize = (800, 600)
+
+    def __init__(
+        self,
+        url: str,
+        digest: str,
+        window_size: Optional[WindowSize] = None,
+        thumb_size: Optional[WindowSize] = None,
+    ):
+        super().__init__(url, digest)
+        self.window_size = (
+            window_size or current_app.config["WEBDRIVER_WINDOW"]["slice"]
+        )
+        self.thumb_size = thumb_size or (800, 600)
```

Comment:
If I didn't understand wrong, screenshots taken with resolution of windows_size and resized to thumb_size(maybe to save space I'm not sure why). I provided thumb_size same with window_size for reports so that they will not shrink.

Reply:
It was hardcoded before. I added to init function so that they can be given as different values

Reply:
OK, thanks :ok_hand: